.github/workflows/sitespeed.yml

```diff
@@ -0,0 +1,47 @@
+name: sitespeed.io
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  run-sitespeed:
+    if: github.event.pull_request.merged == true
+    name: Run sitespeed.io
+    runs-on: ubuntu-20.04
```

Comment:
Can this run on 22.04? Absolutely fine if not

Reply:
Possibly, but this is for 3.2 version only, I'd rather keep 20.04 (which is the version used in the db dump as well - as you can see on line 22).

Reply:
I was thinking that yeah, that makes sense :+1: